test/manager_test.exs

```diff
@@ -29,7 +31,7 @@ defmodule ManagerTest do
 
```

Comment:
The test name doesn't actually describe what is being tested here. Perhaps rename to "max request per minute is respected"